receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -280,6 +285,8 @@ func (jr *jReceiver) startAgent(host component.Host) error {
 	if jr.config.AgentHTTPEndpoint != "" {
 		jr.agentServer = httpserver.NewHTTPServer(jr.config.AgentHTTPEndpoint, &notImplementedConfigManager{}, metrics.NullFactory, jr.settings.Logger)
 
+		jr.settings.Logger.Info("Starting HTTP server for Jaeger Agent", zap.String("endpoint", jr.config.AgentHTTPEndpoint))
+
```

Comment:
```suggestion
```

Don't need the log here, this code is never executed. I am removing it in #36971

Reply:
👍 removing it.